guides/micronaut-metrics-aws/metadata.json

```
@@ -0,0 +1,13 @@
+{
+  "base": "micronaut-metrics",
+  "title": "Collect Metrics with Micronaut and Monitor them on AWS Cloudwatch",
```

```suggestion
  "title": "Collect Metrics with the Micronaut Framework and Monitor them on AWS Cloudwatch",
```

see: https://micronaut.io/brand-guidelines/